ql/src/java/org/apache/hadoop/hive/ql/lib/ExpressionWalker.java

```diff
@@ -18,11 +18,16 @@
 
 package org.apache.hadoop.hive.ql.lib;
 
-import org.apache.hadoop.hive.ql.parse.ASTNode;
-import org.apache.hadoop.hive.ql.parse.HiveParser;
 import org.apache.hadoop.hive.ql.parse.SemanticException;
 
-public class ExpressionWalker extends DefaultGraphWalker {
+import java.util.Stack;
```

Comment:
Consider using `Deque/ArrayDeque` instead, since its usage is recommended over `Stack` class.